### PR TITLE
Bump django to 3.1.10

### PR DESCRIPTION
**Pull request by nicobav**

### CODE REVIEW

**Comment by aprenfors** [comment]
Maybe missing a merge from main or something, since it complains about ical dependency missing.

File: requirements.txt [hunk 1]
```diff
@@ -126,7 +126,6 @@ graphql-core==2.3.2
     #   graphql-relay
 graphql-relay==2.0.1
     # via graphene
-icalendar==4.0.7
```
Comment (nicobav) [inline requirements.txt:129]: ahh, this one got removed by pip-compile

Comment (aprenfors) [reply requirements.txt:129]: Looks like it's my bad, it's missing from req.in 🤦

Comment (nicobav) [reply requirements.txt:129]: I added it there now

**Review by aprenfors — APPROVED** [review]

**Pull request merged**